.nf-core.yml

```diff
@@ -2,14 +2,12 @@ repository_type: pipeline
 lint:
   files_exist:
     - conf/modules.config
-    - conf/test.config
```

Comment:
Can you just remind me what this is about?

Reply:
I restored this file, so we no longer need to ignore that in the linting